Mathlib/Geometry/RingedSpace/OpenImmersion.lean

```diff
@@ -129,7 +129,7 @@ noncomputable def isoRestrict : X ≅ Y.restrict H.base_open :=
 #align algebraic_geometry.PresheafedSpace.is_open_immersion.iso_restrict AlgebraicGeometry.PresheafedSpace.IsOpenImmersion.isoRestrict
 
 @[simp]
-theorem isoRestrict_hom_ofRestrict : H.isoRestrict.hom ≫ Y.ofRestrict _ = f := by
+theorem isoRestrict_hom_ofRestrict : (isoRestrict f).hom ≫ Y.ofRestrict _ = f := by
```

Comment:
Maybe we should change the namespace to enable dot notation as well? @erdOne what do you think?

Reply:
It seems
```lean
noncomputable def _root_.AlgebraicGeometry.PresheafedSpace.Hom.isoRestrict : X ≅ Y.restrict H.base_open :=
```
will not enable dot notation